### ISSUE TITLE
eldoc documentation is incomplete in go-mode

### ISSUE THREAD

**Issue by edkolev** [joaotavora/eglot#97]
The go language server returns
  `func(a ...interface{}) (n int, err error)`,
but the eldoc message which is displayed is
  `func(a ...interface{})`

The issue is probably caused by this `replace-regexp-in-string` here  https://github.com/joaotavora/eglot/blob/master/eglot.el#L1491

To reproduce, run
```elisp
(eglot--sig-info [(:label "func(a ...interface{}) (n int, err error)" :documentation "Println formats using the default formats for its operands and writes to standard output.
Spaces are always added between operands and a newline is appended.
It returns the number of bytes written and any write error encountered.
" :parameters [(:label "a []interface{}")])] 0 0)
```

Or to reproduce in a .go file - place the cursor on Println:
```golang
package main

import (
	"fmt"
)

func Hello(name string) string {
	return "Hello, " + name
}

func Main() {
	fmt.Println(Hello("World"))
}
```

**Comment by mkcms** [comment]
This was fixed in #128.

**Comment by edkolev** [comment]
Awesome, thanks!
